### ISSUE TITLE
Current RBACs not giving proper permissions?

### ISSUE THREAD

**Issue by dougbtv** [istio/istio#364]
## General description

Installing given [official instructions](https://istio.io/docs/tasks/installing-istio.html#installation-steps) is currently failing with the manager pod (and likely ingress & egress pods, however I didn't look at the errors in those pods) crash looping.

The manage pod discovery container complains that `"system:serviceaccount:default:istio-manager-service-account" cannot list istioconfigs.istio.io at the cluster scope.`

## System details

* Istio 0.1.5 via shell install (other issue present @ HEAD currently), or `git clone --branch 0.1.5`
* CentOS 7.3
* Kubernetes 1.6.4

### Further notes

I have had successful installations using [these helm charts](https://github.com/kubernetes/charts/tree/master/incubator/istio) which apparently has somewhat out of date forks of the RBACs. Where the role bindings are still cluster role bindings in those helm chart templates. Which apparently changed in [this commit](https://github.com/istio/istio/commit/e4b6f2eef33341e5061b5599d9ffa0b8ff090bfc).

Unsure if this is a mistaken change in the RBACs or potentially a documentation issue on the installation that I'm incorrectly applying the RBACs.

## Steps to reproduce

```
git clone --branch 0.1.5 --depth 50 https://github.com/istio/istio.git 
cd istio
kubectl apply -f install/kubernetes/istio-rbac-beta.yaml
kubectl apply -f install/kubernetes/istio.yaml
```

## Exact issue encountered

After following above steps to reproduce, the manager pod (as well as ingress & egress pods) are crash looping.

Here's the logs from the discovery container in the manager pod:

```
[centos@kube-master head-istio]$ kubectl logs -f istio-manager-2910860705-95dxq -c discovery
I0606 16:01:21.069186       1 client.go:195] Checking for TPR resources
I0606 16:01:21.107041       1 client.go:205] TPR "IstioConfig" is not ready (User "system:serviceaccount:default:istio-manager-service-account" cannot list istioconfigs.istio.io at the cluster scope. (get IstioConfigs.istio.io)). Waiting...
```

## Work around

I was able to work around this issue with:

```
[centos@kube-master istio]$ git revert e4b6f2eef33341e5061b5599d9ffa0b8ff090bfc
[centos@kube-master istio]$ kubectl apply -f install/kubernetes/istio-rbac-beta.yaml 
[centos@kube-master istio]$ kubectl create -f install/kubernetes/istio.yaml 
```

**Comment by ldemailly** [comment]
that's a fantastic/well done issue report ! thank you 
(cc @thisisnotapril )

I apologize we didn't get to it until now - it's in part because you filled it in the wrong repo
issues report should be in
https://github.com/istio/issues/issues

this being said I think this is fixed in 0.1.6 and in the upcoming 0.2 (though if you can confirm/retry that'd be fantastic)

**Comment by dougbtv** [comment]
Thanks for the compliment, Laurent! I'll give it a whirl with 0.2, also
thanks for the pointer to the proper repo to file issues.
